FRCore/FRCore/Keychain/KeychainService.swift

```diff
@@ -994,7 +994,7 @@ public enum KeychainAccessibility: String {
         case .whenPasscodeSetThisDeviceOnly:
             return String(kSecAttrAccessibleWhenPasscodeSetThisDeviceOnly)
         case .alwaysThisDeviceOnly:
-            return String(kSecAttrAccessibleAlwaysThisDeviceOnly)
+            return String(kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly)
```

Comment:
Have you checked if that breaks any notification or extension stuff? I know we don't do officially extensions but I recall a use case that an extension was used, it might be the authenticator module. As a general idea I like the `kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly` of course as long as we don't break anything unexpectedly

Reply:
I reviewed this with @george-bafaloukas-forgerock. The one I'm using is the most appropriate one as the older one is deprecated. However we need to test this during the release for Push Notifications.